src/main/java/org/jenkinsci/plugins/gogs/GogsProjectProperty.java

```diff
@@ -60,6 +60,9 @@ public String getGogsSecret() {
       }
 
       public JobProperty<?> newInstance(StaplerRequest req, JSONObject formData) throws FormException {
+        if(req == null){
```

Comment:
Not sure why this would be needed, and if so don't think it should be in this PR